clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -152,8 +154,8 @@ abstract class AbstractEurekaSupport {
             throw new RetryableException("Non HTTP 200 response from discovery for instance ${instanceId}, will retry (attempt: $retryCount}).")
           }
         }
-      } catch (RetrofitError retrofitError) {
-        def alwaysSkippable = retrofitError.response?.status == 404
+      } catch (SpinnakerHttpException e) {
```

Comment:
Did I already comment about this elsewhere?  Seems like this needs to be 
```suggestion
      } catch (SpinnakerServerException e) {
```
to retain the previous behavior.